integration/src/test/resources/synchronized-flowref-mule-context-start-config.xml

```diff
@@ -16,7 +16,7 @@
     </flow>
 
     <flow name="flow2">
-        <custom-source class="org.mule.test.routing.SynchronizedFlowRefOnMuleContextStartTestCase$UnblockProcessingSource"/>
+        <test:custom-source class="org.mule.test.routing.SynchronizedFlowRefOnMuleContextStartTestCase$UnblockProcessingSource"/>
```

Comment:
I don't think that the logic of this test class should be in SkeletonSource. What we could do is move the class to org.mule.functional.api.component but since it's only used here i don't see the benefits of doing that.